yt/utilities/exceptions.py

```diff
@@ -14,12 +14,33 @@ def __init__(self, message=None, ds=None):
 
 
 class YTOutputNotIdentified(YTException):
-    def __init__(self, args, kwargs):
+    def __init__(self, filename, args=None, kwargs=None):
+        self.filename = filename
         self.args = args
         self.kwargs = kwargs
 
     def __str__(self):
-        return "Supplied %s %s, but could not load!" % (self.args, self.kwargs)
+        msg = "Could not determine input format from %s" % self.filename
+        if self.args is not None:
+            msg += ", %s" % self.args
+        if self.kwargs is not None:
+            msg += ", %s" % self.kwargs
```

Comment:
```suggestion
        msg =f "Could not determine input format from {self.filename}"
        if self.args is not None:
            msg += f", {self.args}"
        if self.kwargs is not None:
            msg += f", {self.kwargs}"
```
f-strings frenzy!

Reply:
how about running this ?
```
flynt yt.utilities.exceptions
```
(in another PR)